libs/coin-evm/src/signOperation.ts

```diff
@@ -0,0 +1,127 @@
+import { Observable } from "rxjs";
```

Comment:
For some reason this code isn't marked has rename but changed, I guess you do not have any change expect some import right ?

Reply:
Indeed it should be moved, will take a look at that 👀

Reply:
@henrily-ledger 
This is actually just a Github PR ui thing (bug or feature, I don't know).
I assume Github UI displays the "new file" icon above a certain threshold of change in a moved file 🤷‍♂️

But on git side (and Github history), the history is maintained.

When running `git log --follow -- libs/coin-evm/src/signOperation.ts`, all the commits are displayed, including the onces from Kevin when the file was still in LLC.

This is similar to what have been observed here: https://github.com/LedgerHQ/ledger-live/pull/3306#issuecomment-1535988667